website/content/en/docs/golang/monitoring/operator-sdk-monitoring.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Operator SDK monitoring with Prometheus
-linkTitle: Monitoring
+linkTitle: With Prometheus
```

Comment:
to make more clear what it is about.

Reply:
Could you change the title (which will also change the url) to prometheus.md?